react-native/app/components/CommonList.js

```diff
@@ -34,21 +39,22 @@ export default class CommonList extends Component {
                 {
                   text: 'update',
                   onPress: () => this.setState({promptVisible: true, selectedRow: data}),
-                  style: { backgroundColor: 'blue', color: 'white' }
+                  style: {backgroundColor: 'blue', color: 'white'}
                 },
                 {
                   text: 'delete',
                   onPress: () => this.props.onDelete(data),
-                  style: { backgroundColor: 'red', color: 'white' }
+                  style: {backgroundColor: 'red', color: 'white'}
                 }
               ]}>
-              <TouchableWithoutFeedback key={rowID} onPress={() => this.props.onRowPressed(data)}>
+              <TouchableOpacity key={rowID} onPress={() => this.props.onRowPressed(data)}>
```

Comment:
What's the reason for switching from `TouchableWithoutFeedback` to `TouchableOpacity`?